plugin/helpers.go

```diff
@@ -10,6 +10,20 @@ type Helpers interface {
 	// the specifications of the passed bot.
```

Comment:
minor nit: Since you're in this file, mind fixing the typo `ether` on the line above. Should be `either`.